app/models/manageiq/providers/vmware/refresh_helper_methods.rb

```diff
@@ -0,0 +1,21 @@
+module ManageIQ::Providers::Vmware::RefreshHelperMethods
+  extend ActiveSupport::Concern
+
+	def process_collection(collection, key)
+    @data[key] ||= []
```

Comment:
Ah I like the refresh_helper_methods, should make it easier when adding the `NetworkManager`, just need to fix up the indentation